WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/ui/wizards/projectregistration/ProjectCreatedEndingStep.java

```diff
@@ -86,12 +86,22 @@ private void createPageContent(JPanel parent) {
             UserProjectRegistrationWizard wizard = (UserProjectRegistrationWizard) getWizard();
             UserRegistrationStep userRegistrationStep = wizard.userRegistrationStep;
             if (wizard.userWelcomeStep.getRegisterNewId()) {
+                createDebugSurveyInfo(parent);
                 userRegistrationStep.createUserRegistrationSummary(parent);
             }
             createProjectRegistrationSummary(parent);
         }
     }
 
+    /**
+     * Shows the label and link to ask the new user to fill out the survey on
+     * debugging.
+     */
+    private void createDebugSurveyInfo(JPanel parent) {
+        UIUtils.createBoldLabel(parent, "Do you ever debug? Did you know WatchDog now also reports on debugging?");
```

Comment:
@nspruit I accepted this already, but you can extract the string to nl.tudelft.watchdog.core.util.WatchDogGlobals?